tests/dissemination/test_integration.py

```diff
@@ -337,9 +337,9 @@ def test_html_src(host):
     """Submissions with HTML source"""
     # legacy returns 200 with msg: "Unavailable, The author has provided no source to generate PDF, and no PDF."
     resp = requests.get(f"{host}/pdf/cond-mat/9906075v1")
-    assert resp.status_code == 404
+    assert resp.status_code == 500
```

Comment:
This should be a 404.

Reply:
Oops ya

Reply:
Actually i see I'm wrong, this is a case where we really expect there to be a PDF but there isn't one. 500 is fine.